src/Microsoft.ServiceFabric.Actors/Runtime/ActorBase.cs

```diff
@@ -128,7 +128,7 @@ protected virtual Task OnActivateAsync()
         }
 
         /// <summary>
-        ///  Override this method to release any resources. This method is called when actor is deactivated (garbage collected by Actor Runtime).
```

Comment:
@v-rboado  Should be "Override".  A developer would need to override this method.